oteps/XXXX-extending-attributes-to-support-complex-values.md

```diff
@@ -179,47 +162,58 @@ Semantic conventions will be updated with the following guidance
 
 ## Trade-offs and mitigations
 
-1. Allowing arbitrary objects as attributes is convenient but increases the risk of
-   including large, sensitive, mutable, non-serializable, or otherwise problematic
-   data in telemetry.
+### Backends don't support `AnyValue` attributes
 
-   It is RECOMMENDED to enforce `AnyValue` compatibility at the API level.
-   Users and instrumentations SHOULD define custom `AnyValue`-compatible models
-   to minimize misuse and reduce performance overhead.
+While it should be possible to record complex data in telemetry, many backends do not
+support it well - or at all - which can result in individual complex attributes
+or entire telemetry items being dropped or rejected.
```

Comment:
I'm sort of pushing back against this a bit in my latest update:

> The OpenTelemetry Collector already supports transforming data to complex attributes
  via OTTL across all signals, so backends are not free from an obligation to handle
  complex attributes gracefully (even if that just means dropping them currently).

```suggestion
being dropped or rejected.
```